msu/systems/registry/registry_mod_addon.nut

```diff
@@ -0,0 +1,63 @@
+::MSU.Class.RegistryModAddon <- class extends ::MSU.Class.SystemModAddon
+{
+	static GithubURLRegex = regexp("https:\\/\\/github\\.com\\/([-\\w]+)\\/([-\\w]+)");
+	static NexusURLRegex = regexp("https:\\/\\/www\\.nexusmods\\.com\\/battlebrothers\\/mods\\/(\\d+)");
+	__GithubURL = null;
+	__NexusModsURL = null;
+	__UpdateSource = null;
+
+	function getGithubURL()
```

Comment:
The official spelling of GitHub is with `H` being capital. Please fix throughout the PR.